libcxx/test/support/test_allocator.h

```diff
@@ -467,7 +467,10 @@ class limited_allocator {
   TEST_CONSTEXPR_CXX20 pointer allocate(size_type n) { return handle_->template allocate<T>(n); }
   TEST_CONSTEXPR_CXX20 void deallocate(pointer p, size_type n) { handle_->template deallocate<T>(p, n); }
   TEST_CONSTEXPR size_type max_size() const { return N; }
-  TEST_CONSTEXPR BuffT* getHandle() const { return handle_.get(); }
+
+  // In C++11, constexpr non-static member functions are implicitly const, but this is no longer the case since C++14.
```

Comment:
How did you come across this issue?

Reply:
I was working on a draft PR #118141 adding exception tests for other vector operations, which heavily used the `limited_allocator` class and somehow triggered the usage of `operator=`.